### PR TITLE
PXC-4459 Release notes for Percona XtraDB Cluster 8.0.37-29

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0.37-29.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0.37-29.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Percona XtraDB Cluster 8.0.37-29 (2024-0-0)
+
+[Get started with Quickstart Guide for Percona XtraDB Cluster.](../quickstart-overview.md)
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.37-29 (2024-0-0) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.37-29.html).
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.37 and included in Percona XtraDB Cluster are the following:
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: remove this one

File: docs/release-notes/8.0.37-29.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* [PXC-4419](https://perconadev.atlassian.net/browse/PXC-4419): The concurrent drop of users on different cluster nodes resulted in a hanging node.
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): Garbd terminated SST script children after SST was completed.
+
+## Packaging notes
+
+Percona XtraDB Cluster 8.0.37-29 supports Ubuntu 24.04.
+
+
+## Useful links
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: ## Additional resources

File: docs/release-notes/8.0.37-29.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+[Install Percona XtraDB Cluster](../install-index.md)
+
+The [Percona XtraDB Cluster GitHub location](https://github.com/percona/percona-xtradb-cluster)
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: GitHub repository

File: docs/release-notes/8.0.37-29.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: What about
downloads
install
and then GitHub repository

File: docs/release-notes/8.0.37-29.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+* Enhanced management of buffers has been implemented in deleting tablespaces to prevent a possible assertion failure. (Bug #35676106, Bug #36343647)
+
+* Resolved an assertion failure associated with full-text indexes. (Bug #35836581)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.37 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-37.html).
+
+## Bug fixes
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): Percona XtraDB Cluster node stoped when parallel replication workers were executing DDLs via async node.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: Cluster node stopped when
misspelled "stoped"
Also - don't use abbreviations - what is a DDLs? What is async mode (asynchronous)

File: docs/release-notes/8.0.37-29.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+* [PXC-4313](https://perconadev.atlassian.net/browse/PXC-4313): Statements executed in RSU mode generated local GTID events.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: What is RSU?
What is GTID?

File: docs/release-notes/8.0.37-29.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+* [PXC-4363](https://perconadev.atlassian.net/browse/PXC-4363): Concurrent CREATE and DROP USER queries on different nodes resulted in permanent lock.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: CREATE USER and DROP User
resulted in "a" permanent lock.

File: docs/release-notes/8.0.37-29.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+* [PXC-4385](https://perconadev.atlassian.net/browse/PXC-4385): Running a TOI ALTER USER in one node and a SET PASSWORD in another node, resulted in statements hanging  and locking the affected nodes.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: What is TOI?

In the Total Order Isolation mode, an ALTER USER in one mode and a SET PASSWORD in another mode caused statements to hang which locked the nodes.

File: docs/release-notes/8.0.37-29.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: reordered

File: docs/release-notes/8.0.37-29.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 8 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: done

File: docs/release-notes/8.0.37-29.md (hunk 7 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: added

File: docs/release-notes/8.0.37-29.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: rewritten

File: docs/release-notes/8.0.37-29.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: removed

File: docs/release-notes/8.0.37-29.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): Percona XtraDB Cluster node stopped when parallel replication workers were executing Data Definition Language (DDL) commands via asynchronous node.
+
+* [PXC-4313](https://perconadev.atlassian.net/browse/PXC-4313): Statements executed in a mode when operations are not replicated generated local Global Transaction Identifier (GTID) events.
+
+* [PXC-4363](https://perconadev.atlassian.net/browse/PXC-4363): Concurrent CREATE and DROP USER queries on different nodes resulted in a permanent lock.
+
+* [PXC-4385](https://perconadev.atlassian.net/browse/PXC-4385): In the Total Order Isolation mode, an ALTER USER in one mode and a SET PASSWORD in another mode caused statements to hang which locked the nodes.
+
+* [PXC-4397](https://perconadev.atlassian.net/browse/PXC-4397): Both ALTER DATABASE and DROP TRIGGER hung when running these commands simultaneously.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: The instance became unresponsive if ALTER DATABASE and DROP TRIGGER commands were run simultaneously.

File: docs/release-notes/8.0.37-29.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+* [PXC-4401](https://perconadev.atlassian.net/browse/PXC-4401): There was duplicated GU_DBUG_SYNC_WAIT("before_replicate_sync") in galera::ReplicatorSMM::replicate().
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: The function galera::ReplicatorSMM::replicate() contained a duplicate GU_DBUG_SYNC_WAIT("before_replicate_sync") command.

File: docs/release-notes/8.0.37-29.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+* [PXC-4403](https://perconadev.atlassian.net/browse/PXC-4403): Wrong field metadata caused assertion in protocol_classic.cc::store_string().
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: Incorrect field metadata

File: docs/release-notes/8.0.37-29.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+* [PXC-4419](https://perconadev.atlassian.net/browse/PXC-4419): The concurrent drop of users on different cluster nodes resulted in a hanging node.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: A node became unresponsive if DROP USER commands were executed on different nodes.

File: docs/release-notes/8.0.37-29.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): Garbd terminated SST script children after SST was completed.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.37-29.md]: When the State Snapshot Transfer (SST) completed, the Galera Arbitrator Daemon (garbd) also terminated any SST  subprocesses.

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 12 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 11 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 10 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

File: docs/release-notes/8.0.37-29.md (hunk 9 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md]: fixed

**Review by patrickbirch — APPROVED** [review]

File: docs/release-notes/8.0.37-29.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+* [PXC-4397](https://perconadev.atlassian.net/browse/PXC-4397): The instance became unresponsive if ALTER DATABASE and DROP TRIGGER commands were run simultaneously.
+
+* [PXC-4401](https://perconadev.atlassian.net/browse/PXC-4401): The function galera::ReplicatorSMM::replicate() contained a duplicate GU_DBUG_SYNC_WAIT("before_replicate_sync") command.
+
+* [PXC-4403](https://perconadev.atlassian.net/browse/PXC-4403): Incorrect field metadata caused assertion in protocol_classic.cc::store_string().
+
+* [PXC-4419](https://perconadev.atlassian.net/browse/PXC-4419): A node became unresponsive if DROP USER commands were executed on different nodes.
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): When the State Snapshot Transfer (SST) completed, the Galera Arbitrator Daemon (garbd) also terminated any SST subprocesses.
+
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.37-29.md:40]: PXC-4453 is missing

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.37-29.md:40]: Added

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**